packages/event-client/src/event-type.js

```diff
@@ -6,5 +6,7 @@ export const EventType = {
 	Update: 'Update',
 	Delete: 'Delete',
 	Publish: 'Publish',
-	Unpublish: 'Unpublish'
+	Unpublish: 'Unpublish',
+	Success: 'Success',
```

Comment:
Have these new types been agreed with FO or whoever cares about them?

Reply:
I haven't agreed them with anyone, I can run it by FO though. It looks like generally so far Service Bus has been used just to publish data when something changes in the DB so that other services can get in sync. This work might be the first example of publishing an actual message, so none of the existing event types make sense for it.

Reply:
I wonder if we're overloading this type a little, perhaps this success/failure should just be a message property, as it seems quite specific to this use-case. Not quite sure what it's used for though (these event types are separate to the message so that subscribers can filter based on them).

Reply:
Yeah I agree, there is a separate property on the actual event body with the status (success/failure/virus detected). The `sendEvents` function from our event client requires an eventType argument though. I could look at updating it so that that argument is optional.

Reply:
OK looking through to understand a bit more, I think these should just be `Create|Update` events - effectively this is just an update to a `nsip-exam-timetable-submission` entity, where the `status` has changed, so I think that fits. Would you agree?

Reply:
Ok yeah I think that makes sense. I think it should be Create rather than Update though. FO sends us submissions to add to our DB, and we associate them to cases.